raiden/tasks.py

```diff
@@ -27,6 +30,9 @@ def check_version():
             content = requests.get(LATEST).json()
             # getting the latest release version
             latest_release = parse_version(content['tag_name'])
+            security_message = re.search(SECURITY_EXPRESSION, content['body'])
```

Comment:
Can you add some unit tests for the matching of the expression and have some example expression?

Reply:
Ofc, I'll do that.